extra/1-currency-conversion.js

```diff
@@ -15,7 +17,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(brp) {
+  let brl = brp * 5.7;
+  return (Math.round((brl * 0.99) * 100) / 100) ;
```

Comment:
This return function is doing a little too much here. Can you break out the 1% fee calculation into a separate variable and calculate the conversion before you return it?